postcss.config.js

```diff
@@ -1,8 +1,12 @@
 const process = require('process');
+const path = require('path');
 
 module.exports = {
   syntax: 'postcss-scss',
   plugins: {
+    'postcss-jit-props': {
+      files: [path.resolve('node_modules/@rei/cdr-tokens/dist/rei-dot-com/css/cdr-tokens.css')],
```

Comment:
What is this doing? Streamlining the final build output or just there for dev purposes?

Reply:
It's a "just-in-time" plugin which only important css properties being used in the kitchen sink build. This is kind of a POC for the plugin. It doesn't need to be there as it only affects the kitchen sink

Reply:
Ok, I'm going to remove as to not add an extra dependency (plus I want to remove as much postcss as I can/deprecate the `extract-css` from the build in a future update)